app/page.tsx

```diff
@@ -144,15 +174,35 @@ export default function HomePage() {
   const showPaginationButtons = hasMoreMarkets || page !== 1;
 
   return (
-    <div className="justify-center px-6 mt-12 space-y-12 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+    <div className="justify-center px-6 mt-12 space-y-8 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+      <div className="w-full">
+        <h1 className="text-2xl font-semibold text-white capitalize">
+          🔮 {category ? category : "All"}
+        </h1>
+      </div>
       <div className="flex flex-col justify-between w-full gap-5 md:flex-row">
-        <h1 className="text-2xl font-semibold text-white">🔮 All markets</h1>
+        <ToggleGroup
+          value={category}
+          onChange={setCategory}
+          // className="overflow-x-scroll"
+        >
+          <ToogleGroupOption value={""}>All</ToogleGroupOption>
+          {Object.values(Categories).map(category => (
+            <ToogleGroupOption
+              key={category}
+              value={category}
+              className="capitalize"
```

Comment:
Can we add bolder text here? It looks very thin and hard to read.

Reply:
yes. But currently not working, will update `ToogleGroupOption` component to allow it.